homeassistant/components/metoffice/data.py

```diff
@@ -1,78 +1,11 @@
 """Common Met Office Data class used by both sensor and entity."""
 
-import logging
-
-import datapoint
-
-from .const import MODE_3HOURLY
-
-_LOGGER = logging.getLogger(__name__)
-
 
 class MetOfficeData:
```

Comment:
Optional improvement: This could be a dataclass:
https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass

Reply:
To be honest, I'm not sure there would be much benefit from it. I would leave as is.

Reply:
The benefit is the typing and not having to define the init method.

Reply:
Yeah. The problem is that it's kinda tricky to type these things. Datapoint doesn't provide good types for forecast values we use here.